src/frontend/apps/e2e/__tests__/app-impress/common.ts

```diff
@@ -91,6 +92,18 @@ export const addNewMember = async (
   return users[index].email;
 };
 
+export const getGridRow = async (page: Page, title: string) => {
+  const docsGrid = page.getByRole('grid');
+  await expect(docsGrid).toBeVisible();
+  await expect(docsGrid.getByTestId('grid-loader')).toBeHidden();
```

Comment:
This loader is not present directly in the grid, use page. can you do the same for "goToGridDoc" method ? 

```
  await expect(page.getByTestId('grid-loader')).toBeHidden();
```